.travis.yml

```diff
@@ -39,8 +39,6 @@ install:
   # Latest PIP uses wheel by default
   - pip install --upgrade pip
   - pip install $DJANGO
-  - pip install -r requirements.txt
-  - pip install -r requirements_test.txt
```

Comment:
Why are these removed?

Reply:
I believe this was because I was noticing some odd behaviour in testing. I unfortunately cannot recall exactly what was happening, but I believe I was getting tests that passed locally with the `setup.py test` command, but then failing in Travis-CI. Removing these lines resolved the issue and all other tests continue to pass. My Travis-CI history unfortunately has since removed the offending tests from the history.

If I were to make an educated guess, I think we were getting weird version conflicts because we install these packages into the environment and then install things again with the `setup.py test` command.

If we need a definitive answer I can add the lines back and see what happens in testing!

Reply:
I would prefer you take this file straight from master, where CI seems working just fine. :)

Reply:
Updated with requested changes.